src/test/regress/expected/start_stop_metadata_sync.out

```diff
@@ -0,0 +1,244 @@
+
+(1 row)
+
+\d+ distributed_table_4
+                   Table "start_stop_metadata_sync.distributed_table_4"
+ Column |  Type   | Collation | Nullable | Default | Storage | Stats target | Description
+---------------------------------------------------------------------
+ a      | integer |           | not null |         | plain   |              |
+Indexes:
+    "distributed_table_4_a_key" UNIQUE CONSTRAINT, btree (a)
+    "ind1" btree (a)
+
+SELECT * FROM test_view;
+ count
+---------------------------------------------------------------------
+     1
+(1 row)
+
+SELECT * FROM test_matview;
+ count
+---------------------------------------------------------------------
+     1
+(1 row)
+
+SELECT count(*) > 0 FROM pg_dist_node;
+ ?column?
+---------------------------------------------------------------------
+ t
+(1 row)
+
+SELECT count(*) > 0 FROM pg_dist_shard;
+ ?column?
+---------------------------------------------------------------------
+ t
+(1 row)
+
+SELECT count(*) > 0 FROM pg_class WHERE relname LIKE 'distributed_table__' AND relnamespace IN (SELECT oid FROM pg_namespace WHERE nspname = 'start_stop_metadata_sync');
+ ?column?
+---------------------------------------------------------------------
+ t
+(1 row)
+
+SELECT count(*) > 0 FROM pg_class WHERE relname LIKE 'reference_table__' AND relnamespace IN (SELECT oid FROM pg_namespace WHERE nspname = 'start_stop_metadata_sync');
+ ?column?
+---------------------------------------------------------------------
+ t
+(1 row)
+
+\c - - - :worker_1_port
+SET search_path TO "start_stop_metadata_sync";
+-- these should not be found because of stop_metadata_sync_to_node
+SELECT * FROM distributed_table_1;
+ERROR:  relation "distributed_table_1" does not exist
+\d+ distributed_table_4
+SELECT * FROM test_view;
+ERROR:  relation "test_view" does not exist
+SELECT * FROM test_matview;
+ERROR:  relation "test_matview" does not exist
+SELECT count(*) > 0 FROM pg_dist_node;
+ ?column?
+---------------------------------------------------------------------
+ f
+(1 row)
+
+SELECT count(*) > 0 FROM pg_dist_shard;
+ ?column?
+---------------------------------------------------------------------
+ f
+(1 row)
+
+SELECT count(*) > 0 FROM pg_class WHERE relname LIKE 'distributed_table__' AND relnamespace IN (SELECT oid FROM pg_namespace WHERE nspname = 'start_stop_metadata_sync');
+ ?column?
+---------------------------------------------------------------------
+ f
+(1 row)
+
+SELECT count(*) > 0 FROM pg_class WHERE relname LIKE 'reference_table__' AND relnamespace IN (SELECT oid FROM pg_namespace WHERE nspname = 'start_stop_metadata_sync');
+ ?column?
+---------------------------------------------------------------------
+ f
+(1 row)
+
+\c - - - :master_port
+SET search_path TO "start_stop_metadata_sync";
+SELECT * FROM distributed_table_1;
+ col | b
+---------------------------------------------------------------------
+(0 rows)
+
+--cleanup
+DROP SCHEMA start_stop_metadata_sync CASCADE;
```

Comment:
maybe have `SET client_min_messages TO WARNING` to avoid details which might change easily